packages/docusaurus/bin/docusaurus.js

```diff
@@ -21,15 +23,18 @@ const {
   clear,
   writeTranslations,
 } = require('../lib');
-const requiredVersion = require('../package.json').engines.node;
-const pkg = require('../package.json');
-const updateNotifier = require('update-notifier');
-const boxen = require('boxen');
+const {
+  name,
+  version,
+  engines: {node: requiredVersion},
+} = require('../package.json');
 
-// notify user if @docusaurus/core is outdated
+// notify user if @docusaurus packages is outdated
 const notifier = updateNotifier({
-  pkg,
-  updateCheckInterval: 1000 * 60 * 60 * 24, // one day
```

Comment:
Not needed, since it is default value https://github.com/yeoman/update-notifier#updatecheckinterval